src/app/components/Search/Search.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import SearchForm from './SearchForm';
+import SearchResultsList from './SearchResultsList';
+import * as searchActions from '../../actions/SearchActions';
+
+class Search extends React.Component {
+  constructor(props) {
+    super(props);
+    const { dispatch } = props;
+    this.state = props;
```

Comment:
You're not using the state anywhere  in this function it seems? Unless if you need to initialize it here so that the `mapStateToProps` function works?

Reply:
Removing this line as it is superfluous